File: Become 1 Star/Golang/id_and_ship.go
package main
import "fmt"

func main(){
	// your code goes here
	var t int
	var c string
	fmt.Scan(&t)
	
	for t>0{
	    fmt.Scan(&c)
	   
	        if c == "B" || c == "b" {
	            fmt.Println("BattleShip")
	        }
	        if c == "C" || c == "c" {
	            fmt.Println("Cruiser")
	        }
	        if c == "D" || c == "d" {
	            fmt.Println("Destroyer")
	        }
	        if c == "F" || c == "f" {
	            fmt.Println("Frigate")
	        }
	      
	    t--
	}
}

File: Become 1 Star/Java/id_and_ship.java
import java.util.Scanner;

public class id_and_ship {
        public static void main (String[] args) throws java.lang.Exception
        {
            try (// your code goes here
            Scanner scn = new Scanner(System.in)) {
                int t = scn.nextInt();
                
                for(int i = 0; i < t; i++){
                    char ch = scn.next().charAt(0);
                    
                    if(ch == 'B' || ch == 'b'){
                        System.out.println("BattleShip");
                    }else if(ch == 'C' || ch == 'c'){
                        System.out.println("Cruiser");
                    }else if(ch == 'D' || ch == 'd'){
                        System.out.println("Destroyer");
                    }else
                    System.out.println("Frigate");
                }
            }
        }
}
